cmd/admin-service-status.go

```diff
@@ -51,10 +51,29 @@ EXAMPLES:
 `,
 }
```

Comment:
Can you have a commit message concise i.e it doesn't not wrap around.  and also have a 50-70 column width so that the commit messages are warping. 

Best style is to observe linux kernel commit logs.